.github/workflows/snapshot.yml

```diff
@@ -12,6 +12,11 @@ jobs:
     runs-on: ubuntu-latest
     name: snapshot
     if: ${{github.event.pull_request.merged == true}}
+    services:
+      registry:
+        image: registry:2
+        ports:
+          - 5000:5000
```

Comment:
Ah, sorry. Since snapshot already pushes to quay, we shouldn't need the local registry or the `driver-opts` on the `setup-buildx-action`.

Reply:
I think later we can perhaps use the local registry and then copy them all at once if everything works. Not perfect, but slightly more "atomic" than what we have now.